components/tools/OmeroWeb/test/integration/test_scripts.py

```diff
@@ -49,11 +49,13 @@ def upload_script(self):
 
         script = """
 import omero
-from omero.rtypes import rstring, rint, wrap
+from omero.rtypes import rstring, rint, wrap, robject
```

Comment:
maybe time to order alphabetically the import

Reply:
adding

```
[flake8]
import-order-style = smarkets
```

to `.flake8` would be one method to enforce the imports. see https://pypi.org/project/flake8-import-order/ for more info.

Reply:
I'm not sure that flake8 would catch these imports since they are in a script string.